### ISSUE TITLE
Build failure with Stackage

### ISSUE THREAD

**Issue by snoyberg** [serokell/log-warper#87]
```
[13 of 19] Compiling System.Wlog.IOLogger ( src/System/Wlog/IOLogger.hs, dist/build/System/Wlog/IOLogger.o )

src/System/Wlog/IOLogger.hs:173:34: error:
    • Couldn't match type ‘Map LoggerName Logger’ with ‘LoggerName’
      Expected type: Element [LoggerName] -> LogTree -> LogTree
        Actual type: LogTree -> LoggerName -> LogTree
    • In the first argument of ‘foldl'’, namely ‘addLoggerToTree’
      In the expression: foldl' addLoggerToTree lt xs
      In an equation for ‘createLoggers’:
          createLoggers xs lt = foldl' addLoggerToTree lt xs
    |
173 |     createLoggers xs lt = foldl' addLoggerToTree lt xs -- Add logger to tree
    |                                  ^^^^^^^^^^^^^^^
```

**Comment by chshersh** [comment]
@snoyberg Thanks for reporting this issue! It should be fixed in version `1.8.4`.

**Comment by snoyberg** [comment]
Cool, thanks!
